source/en/api_nearest_neighbor.rst

```diff
@@ -32,6 +32,21 @@ We show each field below:
    Specify parameters for the algorithm.
    Its format differs for each ``method``.
 
+   common
+     :threads:
+        The number of threads execute random_projection and search.
+        The bigger it is, query latency becomes smaller because of data divided and excuted by multi thread in parallel.
+        This option added since 0.9.0. Previously 0.8.9, it worked with 1 thread.
+        (Integer)
+
+        * Range: ``threads`` < 0  ``threads`` regard as The number of logical cores of CPU
```

Comment:
How about writing the explanation in the next line, like below?

The behavior of this option varies as below:
* ``threads`` < 0 
  * The number of threads is set to the number of logical CPU cores.
* ``threads`` = 0 
  * The number of threads is set to 1. 
* 1 <= ``threads `` <= The number of logical CPU cores
  * The number of threads is set to ``threads``